connect/src/test/scala/org/apache/spark/sql/connect/planner/SparkConnectPlannerSuite.scala

```diff
@@ -58,11 +58,6 @@ class SparkConnectPlannerSuite extends SparkFunSuite with SparkConnectPlanTest {
 
   protected var spark: SparkSession = null
 
-  override def beforeAll(): Unit = {
-    super.beforeAll()
-    TestUtils.configTestLog4j2("INFO")
```

Comment:
`TestUtils.configTestLog4j2` always print log to console, some noises. I So I think it is better not to use this method unless debugging for something.

Reply:
Yeah, I agree. Is this the only place to fix?

Reply:
Not limited to the `connect` module?